### ISSUE TITLE
Psalm warning re closure passed to usort

### ISSUE THREAD

**Issue by JakeQZ** [MyIntervals/emogrifier#1212]
```xml
    <InvalidArgument>
      <code>\usort(
            $cssRules['inlinable'],
            /**
             * @param array{selector: string, line: int} $first
             * @param array{selector: string, line: int} $second
             */
            function (array $first, array $second): int {
                return $this-&gt;sortBySelectorPrecedence($first, $second);
            }
        )</code>
    </InvalidArgument>
```

ref #1211

**Comment by JakeQZ** [comment]
Might be that we need angle brackets rather than curlies for the `@param`, but I could be "barking up the wrong tree" with that suggestion.

**Comment by JakeQZ** [comment]
I saw a different error regarding the `usort` recently, but can't reproduce it now.  See #1223.

Closing as duplicate.  Whatever ths issue is is covered by the more generic #1223.

**Comment by JakeQZ** [comment]
Re-opened.  This is specifically silenced in the `baseline`.

**Comment by JakeQZ** [comment]
https://github.com/vimeo/psalm/issues/10188

**Comment by JakeQZ** [comment]
Fixed by #1232.  Closing.
